src/OrchardCore.Modules/OrchardCore.Html/Handlers/HtmlBodyPartHandler.cs

```diff
@@ -13,7 +13,8 @@ public class HtmlBodyPartHandler : ContentPartHandler<HtmlBodyPart>
     {
         private readonly ILiquidTemplateManager _liquidTemplateManager;
         private readonly HtmlEncoder _htmlEncoder;
-        private HtmlString _bodyAspect;
+        private IHtmlContent _bodyAspect;
```

Comment:
please change back to `HtmlString`

Reply:
You could try it, it will raise invalid casting issue

Reply:
Revert the change, it was a weird error ;)